packages/datatrak-web/src/components/Button.tsx

```diff
@@ -32,7 +32,7 @@ const ButtonWrapper = ({
   if (!tooltip) return children;
   return (
     <Tooltip title={tooltip} arrow enterDelay={1000}>
-      <span>{children}</span>
```

Comment:
I probably should have put a comment but this is needed to fix a js console error where MUI-tooltips complain that disabled buttons cannot have a tooltip

Reply:
Oh right, I removed because it was causing some weird styling of the grid items